explorer/interpreter/value.h

```diff
@@ -1421,62 +1420,42 @@ class AssociatedConstant : public Value {
 };
 
 // A first-class continuation representation of a fragment of the stack.
-// A continuation value behaves like a pointer to the underlying stack
-// fragment, which is exposed by `Stack()`.
+// The representation of a continuation is opaque and determined by the
+// interpreter.
 class ContinuationValue : public Value {
  public:
-  class StackFragment {
+  // The representation of a continuation is derived from this.
```

Comment:
Is the representation always going to be StackFragment? I think it may be clearer to explain the rationale behind this. e.g., instead of saying "The representation ... is opaque", instead call it "StackFragmentInterface" and say "// Implemented by the interpreter's StackFragment. This is an interface to allow for layering between the AST and interpreter."

Reply:
I think `StackFragmentInterface` would end up being a confusing name, because this doesn't actually have the interface for a stack fragment on it (the interactions with the action stack all live in `StackFragment` itself), but I've done some rephrasing and expansion of the comment to point at `StackFragment` and say we don't expect other derived classes.

Reply:
SGTM